client-side/src/stories/verticalTabs/verticalTabss.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import './verticalTabs.scss'
+const  VerticalTabs=({ labels=['tab1','tab2'],elements=[<p>tab 1</p>, <pp>tab 2</pp>]})=> {
```

Comment:
you shouldn't use hard-code for defaultValues.
more of this, those props are marked as isRequired so don't need defaultValues